.github/workflows/lint.yaml

```diff
@@ -13,13 +13,18 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: install go
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v3
         with:
-          go-version: 1.17
```

Comment:
We need at least 1.18 in the CI to install and run go vuln check.

Reply:
Should we use go 1.19 instead?

Reply:
IMO we'd stay as close as possible to the minimum go version expressed in the go.mod https://github.com/auth0/go-jwt-middleware/blob/e00e3d716644faff0502846652343c5f40d4e1d2/go.mod#L3

Also I wouldn't bump the version in the go mod yet as 1.17 is still fully supported by google (they support the last 2 released versions before the latest, e.g. right now we are on 1.19 so 1.18 and 1.17 are still supported).

Reply:
Consider having a matrix for building in all supported Go versions (not linting/checking/etc, just building).